notebook/services/api/api.yaml

```diff
@@ -814,6 +814,10 @@ definitions:
         type: string
         description: Last modified timestamp
         format: dateTime
+      size:
+        type: integer
+        description: "The size of the file or directory."
+        format: bytes
```

Comment:
I think this should just go in the description. Swagger specifies a fixed list of formats for things like `dateTime`. From the spec's point of view, this is just a number - I don't think it has a way of declaring a unit.